cromwell-drs-localizer/src/main/scala/drs/localizer/CommandLineParser.scala

```diff
@@ -14,15 +14,25 @@ class CommandLineParser extends scopt.OptionParser[CommandLineArguments](Usage)
 
   head("cromwell-drs-localizer", localizerVersion)
 
-  arg[String]("drs-object-id").text("DRS object ID").required().
+  arg[String]("drs-object-id").text("DRS object ID").optional().
     action((s, c) =>
       c.copy(drsObject = Option(s)))
-  arg[String]("container-path").text("Container path").required().
+  arg[String]("container-path").text("Container path").optional().
     action((s, c) =>
       c.copy(containerPath = Option(s)))
-  arg[String]("requester-pays-project").text("Requester pays project").optional().
+  arg[String]("requester-pays-project").text(s"Requester pays project (only valid with '$Google' auth strategy)").optional().
     action((s, c) =>
       c.copy(googleRequesterPaysProject = Option(s)))
+  opt[String]('m', "manifest-path").text("File path of manifest containing multiple files to localize").
+    action((s, c) =>
+      c.copy(manifestPath = Option(s)))
+  opt[String]('r', "requester-pays-project").text(s"Requester pays project (only valid with '$Google' auth strategy)").optional().
+    action((s, c) => {
+      c.copy(
+        googleRequesterPaysProject = Option(s),
+        googleRequesterPaysProjectConflict = c.googleRequesterPaysProject.exists(_ != s)
```

Comment:
scopt doesn't allow adding an `action` to a `checkConfig` (even though it compiles). It also doesn't allow a `failure` from an `action`. 🙄 This is the best I could come up with to fail if both are specified and don't match. I 🤣 when I thought of it, but I wasn't totally offended by the resulting code when I tried it.

Reply:
Ha, I like it, assuming we are confident that these are always processed in the written order.